<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightDirection</class>
 <widget class="QWidget" name="LightDirection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>149</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>131</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="XDirLight">
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="YDirLight">
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ZDirLight">
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
